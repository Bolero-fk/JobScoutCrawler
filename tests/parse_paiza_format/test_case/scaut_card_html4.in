<div class="MessageCardHeader"><img class="ScoutImage" src="sample4.jpg" alt="スカウト一覧画像"></div><div class="MessageCardBody"><div class="MessageCardTitle"><div class="MessageCardTitleName">PQRコンサルティング株式会社</div><div class="MessageUnReadPoint" style="display: none;"></div></div><div class="MessageCardText">テスト説明4テスト説明4テスト説明4テスト説明4</div><ul class="MessageCardStatus"><li class="MessageCardStatusItem"><i class="fas fa-yen-sign"></i>  200万 〜 500万円</li><li class="MessageCardStatusItem"><i class="fas fa-map-marker-alt"></i>  神奈川県</li><li class="MessageCardStatusItemSub"><i class="fas fa-code"></i> C C++ C#</li></ul><div class="MessageCardDate"><span style="margin-right: 0.25rem;">2023/04/25</span><span style="color: rgb(46, 173, 67);">残り13日</span></div></div>