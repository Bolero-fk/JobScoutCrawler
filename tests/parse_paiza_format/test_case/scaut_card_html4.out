{"company_name": "PQRコンサルティング株式会社", "description": "テスト説明4テスト説明4テスト説明4テスト説明4", "min_salary": 200, "max_salary": 500, "location": "神奈川県", "using_lang": "C C++ C#", "recieve_date": "2023/04/25", "remaining_days": 13}